helm/cert-exporter/templates/daemonset.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: cert-exporter
+  namespace: {{ .Values.namespace }}
+  labels:
+    app: cert-exporter
+spec:
+  updateStrategy:
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      app: cert-exporter
+  template:
+    metadata:
+      name: cert-exporter
+      labels:
+        app: cert-exporter
+      annotations:
+        releasetime: {{ $.Release.Time }}
+    spec:
+      tolerations:
+      # Tolerate all taints for observability
+      - operator: "Exists"
+      serviceAccountName: cert-exporter
+      priorityClassName: giantswarm-critical
+      containers:
+      - name: cert-exporter
+        image: "{{ .Values.image.registry }}/{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        args:
+        {{ if (.Values.Installation) }}
+        - --cert-paths={{ .Values.Installation.V1.Monitoring.CertExporter.CertPath }}
+        {{ else }}
+        - --cert-paths=/etc/kubernetes/ssl
```

Comment:
The `.Values.Installation` will only work in the control plane because it comes from the draughtsman values configmap.

The else block should still be fine with tenant clusters but please make sure this is tested.